docs/concepts.md

```diff
@@ -66,7 +66,9 @@ Components can be as simple as a single Kubernetes resource (e.g. a Deployment)
 
 * Typically, **you autogenerate it** with the [`ks generate`](/docs/cli-reference/ks_generate.md) command. In this case, the manifest is expressed in a language called [Jsonnet](#jsonnet).
 
-* Alternatively, you can **manually drop in** a file. In this case, your manifest can be any one of YAML, JSON, or Jsonnet. Because all three languages are supported, *this approach allows you to introduce ksonnet to existing codebases without significant rewrites*.
+* Alternatively, you can **manually drop in** a file into `components/`. In this case, you can use either JSON or Jsonnet, as long the file is saved with the `*.jsonnet` extension. (JSON is a subset of Jsonnet so no other changes are needed. YAML is not currently supported, but you can use an open-source CLI tool such as [yaml2json](https://github.com/bronze1man/yaml2json) to convert from YAML to JSON).
+
+  *This approach allows you to introduce ksonnet to existing codebases without significant rewrites, although better tooling may be added in the future*.
```

Comment:
I'm questioning use of the term `although`. Can we still say thing with out this clause? 

*This approach allows you to introduce ksonnet to existing codebases.*